README.md

```diff
@@ -18,19 +18,30 @@ wash build
 
 ## Run
 
-Prerequisites:
+### Prerequisites
 
 - [wash 0.29](https://wasmcloud.com/docs/installation) or later
 - A built couchbase capability provider, see [#build](#build)
 - Setup Couchbase server with the required configuration for testing using docker-compose.yaml in the repo.
-   ```
-   docker-compose up -d
-   ```
-  Alternatively, you can use [Quick Install](https://docs.couchbase.com/server/current/getting-started/do-a-quick-install.html) guide with a bucket named **test** created.
- 
+
+```bash
+docker-compose up -d
+```
+
+Alternatively, you can use [Quick Install](https://docs.couchbase.com/server/current/getting-started/do-a-quick-install.html) guide with a bucket named **test** created.
+
+### Running
 
 ```shell
 wash up -d
+# TODO: provide encryption key setup
```

Comment:
Another note to self: will need to instruct on how to use the secrets-nats-kv backend as well here, specifically how to install